### PR TITLE
bug-fix: patched dir owned by runner

### PR DESCRIPTION
**Pull request by alex-mozejko**
in relation to:

https://github.com/summerwind/actions-runner-controller/issues/283

### CODE REVIEW

**Comment by alex-mozejko** [comment]
commit e719724 added so that images are always built using the latest possible actions/runner release. 

could be nice to build this daily so no need to keep bumping the version

File: runner/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,6 @@
 FROM ubuntu:18.04
 
 ARG TARGETPLATFORM
-ARG RUNNER_VERSION=2.274.2
```
Comment (mumoshu) [inline runner/Dockerfile:4]: Thanks for the fix! But could we keep this build-arg as this is required to inject the runner version from within our github actions workflow?

Comment (mumoshu) [reply runner/Dockerfile:4]: > could be nice to build this daily so no need to keep bumping the version

Sorry I missed your comment. I got your intention, but doing so would prevent us from tagging container images with their RUNNER_VERSION.

I'd certainly investigate a way to automate it, probably reusing your snippet for getting the latestest available runner version. But it should be done in a separate PR!

**Review by mumoshu — APPROVED** [review]
LGTM on the `--chown` addition. Also, please allow me to replicate the same fix onto the dockerd-within-runner-container variant of your Dockerfile too!

Thanks for your contribution, @alex-mozejko ☺️

**Pull request merged**